### DISCUSSION TITLE
Do you have to migrate to Paste v1 before 2.0?

### DISCUSSION THREAD

**Discussion by kyliepedersen** [twilio-labs/paste#2818] [Q&A] [answered]
Hello!

I am an engineer on Marketing Campaigns and we are starting discussion on migrating to Paste (woo!). I just had some clarifying questions around two different versions of Paste I have heard about recently. My understanding is that Paste 2.0 is not yet released? MC does not yet have a timeline for migration, but we are hoping some time next year. A couple questions:

1. Is it a requirement to migrate to the first version of Paste before moving to 2.0?
2. Do you folks have a tentative timeline on the 2.0 release?

Thank you!

**Comment by SiTaggart** [comment, marked as answer]
Hi @kyliepedersen 

Great questions! Let me start by clarifying something, there is no v1 or v2 of Paste. It's just Paste.

What we will be doing is adding a new Theme that you'll be able to select with a new set of design tokens that will change the look and feel of the components. It will sit alongside the existing Themes.

Coupled with that work, we will be incrementally updating the design of some components to better utilized some of the improvements we're making to the visual design language.

We practice continuous deployment, so you will see those updates come week by week over the first two quarters of 2023. 

We're aiming for the new theme to be in Paste by Q1, and any component redesigns should be complete in Paste by end of Q2 (maybe sooner). 

What that means is, you can start adopting Paste _today_, just like SendGrid Email Platform have been doing for the past year or so, and as you migrate incrementally, you'll see improvements and changes as you update your Paste version. The idea is to allow incremental adoption without impacting any feature work or timelines.

What's most important to highlight, this isn't a "do it twice" migration. If you start adopting Paste now, you won't have to "do it again" later. Just keep your Paste version up to date as we release new features and the migration to the new design language will be effortless.

**Comment by kyliepedersen** [comment]
Fabulous, thank you for the clarification! This is great. We're so excited to start using Paste!! Appreciate the help.
